### DISCUSSION TITLE
more than 200mpbs hevc can not play on the IOS device

### DISCUSSION THREAD

**Discussion by huahuasousou** [photoprism/photoprism#2854] [Q&A]
I found that h265/hevc files larger than 200mpbs cannot be played on ios devices. I tried with iphone13pro and iphone13pro max, ipad pro 12.9 2018 . When it is set to 150mbps or less then there is no problem.
I confirmed that I don't have ffmpeg turned on for conversion and that PHOTOPRISM_FFMPEG_BITRATE is set to 500. This doesn't trigger a cast.

you can find the hevc file in here:
https://drive.google.com/file/d/1xJt9XIq-N1rXnZp2kFilFlCH_GtvYUqS/view?usp=sharing

**Comment by graciousgrey** [comment]
Might be a limitation of your devices or network?
